kuksa_databroker/databroker/src/main.rs

```diff
@@ -247,6 +248,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .value_parser(clap::builder::NonEmptyStringValueParser::new())
                 .required(false),
         )
+        .arg(
+            Arg::new("jwt-public-key")
+                .display_order(5)
+                .long("jwt-public-key")
```

Comment:
In the C++ version we use --insecure and then specifying the public key; same in kuksa client. I do not know if it is needed to have consistency. But in my first try I thought I need to use --insecure=false as 'always'.

Reply:
`--insecure` specifies whether to allow non-encrypted communication, i.e. not using TLS. It's unrelated to authorization.

Reply:
Yeah, you're right. It would be:   '--cert-path arg (=".")                [mandatory] Directory path where 
                                        'Server.pem', 'Server.key' and 
                                        'jwt.key.pub' are located.' in the kuksa-val-server. I mingeld things up my bad.